src/System.Text.Json/tests/JsonReaderTests.cs

```diff
@@ -1,5 +1,6 @@
 ﻿using System.Collections.Generic;
```

Comment:
For now let's just throw. But still there should be a test validating that it throws.